internal/pkg/cli/cli.go

```diff
@@ -129,12 +128,14 @@ func indentListItem(multiline string) string {
 	return strings.Join(prefixedLines, "\n")
 }
 
-func quoteStringSlice(in []string) []string {
-	quoted := make([]string, len(in))
+type stringMutator func(string) string
+
+func mutateStringSlice(in []string, mutate stringMutator) []string {
```

Comment:
I feel like maybe this is one of the occasions where generics are OK, what do you think of:
```go
func applyAll[T any](in []T, fn func(item T) T) []T {
	out := make([]T, len(in))
	for i, v := range in {
		out[i] = fn(v)
	}
	return out
}